app/controllers/application_controller.rb

```diff
@@ -16,6 +16,14 @@ class PageNotFound < StandardError; end;
   before_action :check_for_pdf
   after_action :store_location
 
+  NO_REDIRECT = ["/users/sign_in",
+    "/users/sign_up",
+    "/users/password/new",
+    "/users/password/edit",
+    "/users/confirmation",
+    "/users/sign_out"
+  ].freeze
```

Comment:
I wonder if we should get rid of these as I don't think users are still able to log in.
This is something that precedes me as well but looking at the current website those pages look a bit broken :)

Reply:
Didn't know it was a holdover, and yes I know :)